src/signals/incident/components/form/TextareaInput/index.js

```diff
@@ -8,13 +8,16 @@ const TextareaInput = ({ handler, touched, value, hasError, meta, parent, getErr
   meta?.isVisible && (
     <Header meta={meta} options={validatorsOrOpts} touched={touched} hasError={hasError} getError={getError}>
       <TextArea
+        id={meta.id}
```

Comment:
Instead of assigning an id to every input, we could also use the `meta.name` property which is derived from the key of the question in the form generator. For example, the 'description' question in wizard-step-1 already passes `meta: {name: 'description'}` to the TextArea component. 

This would also prevent issues where someone forgets to add the 'id' field to the meta key of a question.

Reply:
Good suggestion! I'll use that!